xml/upgrade-to-pacific.xml

```diff
@@ -0,0 +1,222 @@
+   <title>Verifying access to software repositories and container images</title>
+   <para>
+    Verify that each cluster node has access to the &cephos; and &productname;
+    &productnumber; software repositories, as well as the registry of container
+    images.
+   </para>
+   <sect3 xml:id="verify-previous-upgrade-patch-repos-repos-7p">
+    <title>Software repositories</title>
+    <para>
+     If all nodes are registered with SCC, you will be able to use the
+     <command>zypper migration</command> command to upgrade. Refer to
+     <link
+      xlink:href="https://documentation.suse.com/sles/15-SP3/html/SLES-all/cha-upgrade-online.html#sec-upgrade-online-zypper"/>
+     for more details.
+    </para>
+    <para>
+     If nodes are <emphasis role="bold">not</emphasis> registered with SCC,
+     disable all existing software repositories and add both the
+     <literal>Pool</literal> and <literal>Updates</literal> repositories for
+     each of the following extensions:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       SLE-Product-SLES/15-SP3
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       SLE-Module-Basesystem/15-SP3
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       SLE-Module-Server-Applications/15-SP3
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       SUSE-Enterprise-Storage-7.1
+      </para>
+     </listitem>
+    </itemizedlist>
+   </sect3>
+   <sect3 xml:id="verify-previous-upgrade-patch-repos-images-7p">
+    <title>Container images</title>
+    <para>
+     All cluster nodes need access to the container image registry. In most
+     cases, you will use the public SUSE registry at
+     <literal>registry.suse.com</literal>. You need the following images:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       registry.suse.com/ses/7.1/ceph/ceph
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       registry.suse.com/ses/7.1/ceph/grafana
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       registry.suse.com/ses/7.1/prometheus/prometheus-server
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       registry.suse.com/ses/7.1/prometheus/prometheus-node-exporter
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       registry.suse.com/ses/7.1/prometheus/prometheus-alertmanager
+      </para>
+     </listitem>
+    </itemizedlist>
+    <para>
+     Alternatively&mdash;for example, for air-gapped
+     deployments&mdash;configure a local registry and verify that you have the
+     correct set of container images available. Refer to
+     <xref
+     linkend="deploy-cephadm-configure-registry"/> for more details
+     about configuring a local container image registry.
+    </para>
+   </sect3>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="upgrade-cluster-nodes">
+  <title>Migrate &sls; on each cluster node to version &cephos;</title>
+
+  <para>
+   EMPTY
```

Comment:
should there be content here?

Reply:
@mgfritch How to update the underlying SLES? Via `zypper migrate`?

Reply:
This depends on if the user is using SMT, SCC, or another means. Can we ref the SLES15SP3 documentation here?

Reply:
Yes, do. Let's not repeat instructions.